_data/capabilities.yaml

```diff
@@ -13,6 +13,7 @@
     - messagingWindowAPI: "Supported"
     - connectorAPI: "Supported"
     - weChat: "N/A in Channel"
+    - viber: "N/A in Channel"
```

Comment:
@joshuajott are we using this data anywhere? I think capabilities list moved to KC

Reply:
reverted.